### PR TITLE
Task 2 Add zombies to page

### PR DESCRIPTION
**Pull request by ReactorV**
image

### CODE REVIEW

File: Task2/css/app.css [hunk 1]
```diff
@@ -0,0 +1,61 @@
+*,
+*:before,
+*:after {
+    box-sizing: border-box;
+}
+
+.zombies {
+  display: flex;
```
Comment (ExSamaelSolbeg) [inline Task2/css/app.css]: two spaces

File: Task2/js/app.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombie = document.createElement('div');
+
+        zombie.classList.add('zombie');
+
+        zombie.addEventListener('click', () => {
+            zombie.classList.remove(zombie.classList[zombie.classList.length - 1]);
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: wtf?
you should add "zombie" class by default on zombieElement
then you should add "dead" class on click

File: Task2/js/app.js [hunk 3]
```diff
@@ -0,0 +1,24 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombie = document.createElement('div');
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: zombie => zombieElement

File: Task2/js/app.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+
+        zombie.classList.add('zombie');
+
+        zombie.addEventListener('click', () => {
+            zombie.classList.remove(zombie.classList[zombie.classList.length - 1]);
+            zombie.classList.add('dead');
+        });
+
+        if (zombies[i]['type'] === ZOMBIE_TYPE.SMALL) {
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js]: zombies[i]['type']
really?

not just 
zombies[i].type
?

what's the point of using ['']
?

File: Task2/css/app.css [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+
+.mad {
+  background-image: url("../icons/zombie-mad.gif");
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  padding: 40px 0 0 40px;
+ }
+
+.mad:hover {
+  cursor: pointer;
```
Comment (ExSamaelSolbeg) [inline Task2/css/app.css]: why can't you just use "cursor property" on ".mad" class?

File: Task2/css/app.css [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+.small {
+  background-image: url("../icons/zombie-small.gif");
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  padding: 90px 40px 0 50px;
+}
+
+.small:hover {
+  cursor: pointer;
```
Comment (ExSamaelSolbeg) [inline Task2/css/app.css]: same here

File: Task2/css/app.css [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+
+.strong {
+  background-image: url("../icons/zombie-strong.gif");
+  background-repeat: no-repeat;
+  background-origin: content-box;
+  padding-top: 40px;
+}
+
+.strong:hover {
+  cursor: pointer;
```
Comment (ExSamaelSolbeg) [inline Task2/css/app.css]: same here

File: Task2/css/app.css [hunk 8]
```diff
@@ -0,0 +1,61 @@
...
+}
+
+.dead {
+  background-image: url("../icons/zombie-dead.gif");
+  background-repeat: no-repeat;
+  background-origin: content-box;
```
Comment (ExSamaelSolbeg) [inline Task2/css/app.css]:
```
background-repeat: no-repeat;
background-origin: content-box;

```
repeated 3 times
why can't you just move this styles into ".zombie" class?

**Review by ExSamaelSolbeg — CHANGES_REQUESTED** [review]

**Comment by ExSamaelSolbeg** [comment]
your branch was not set up
you can merge PR without any approve
take a look at the link which I sent you before to set up merging policy

File: Task2/js/app.js (hunk 4 above)
Comment (VVadzim) [reply Task2/js/app.js]: as for me it’s more convenient to use either only square brackets or only dots, where it is possible)) but I agree with you that zombies[i].type => requires less characters

File: Task2/js/app.js [hunk 9]
```diff
@@ -0,0 +1,23 @@
+document.addEventListener('DOMContentLoaded', () => {
+    const container = document.querySelector('.zombies');
+
+    for (let i = 0; i < zombies.length; i++) {
+        const zombieElement = document.createElement('div');
+
+        zombieElement.classList.add('zombie');
+
+        zombieElement.addEventListener('click', () => {
+            zombieElement.classList.add('dead');
```
Comment (ExSamaelSolbeg) [inline Task2/js/app.js:10]: well... this solution is close to completed
but I guess if you click on the first zombie the last one will dead

Comment (ReactorV) [reply Task2/js/app.js:10]: ? but last zombie is alive... why he'll die if every zombieElement have his own addEventListener?

**Review by ExSamaelSolbeg — APPROVED** [review]

**Pull request merged**